### ISSUE TITLE
PHP warning messages on many pages

### ISSUE THREAD

**Issue by KevinLi19** [ProjectNami/projectnami#387]
We got a similiar issue on our Azure website. (https://github.com/ProjectNami/projectnami/issues/338)
There are many warning messages on the top of many pages, even we did all the steps suggested for usual website.  For example: 
'Warning: Invalid argument supplied for foreach() in D:\home\site\wwwroot\wp-includes\translations.php on line 1283',
'Warning: Invalid argument supplied for foreach() in D:\home\site\wwwroot\wp-includes\translations.php on line 1476'

We verified that the following 3 plugins add more or less warnings:
Events Manager 
iThemes Security 
PressPermit Pro

**Comment by patrickebates** [comment]
The renaming of the parsed_types file from the other ticket made no difference for you either?

**Comment by KevinLi19** [comment]
> The renaming of the parsed_types file from the other ticket made no difference for you either?

Sorry but that's not working. Changed /wp-includes/fields_map.parsed_types.php  to fields_map.parsed_types.rename.php. And got a new same name file generated. 

Thanks!

**Comment by patrickebates** [comment]
Yes, a new file with almost no data would be generated.  Normally that resolves the issue, as the cause is a poorly formatted array within the file.

Will continue looking into this.
